en/docs/debugging/debug.md

```diff
@@ -0,0 +1,258 @@
+# Debug Ballerina Programs
+
+When writing large-scale applications with complex logic, it is quite possible to have syntax, semantic,
```

Comment:
`it is quite possible to have syntax` - this sounds like it is a good thing to have syntax, semantic, or runtime issues in your code. Shall we reword this? @praneesha

Reply:
added temporary commit suggestion.
Will improve this from another PR